### ISSUE TITLE
Update to implement discordjs v14

### ISSUE THREAD

**Issue by Nispeon** [EdouardCourty/basic-discordbot-template#1]
Hello there,
You may not remember me but I was in a class you taught discord js to earlier this year.

I've since started working on my own bot project (which I hope will grow big) and it uses your template as a basis.
Granted I've changed a ton of things around it but the base is still here, and I've come to update it to implement the newer version of the `discordjs` library to be able to use slash commands. They're more accessible and Discord has really been pushing lately to make them a standard.

TL;DR : Would you be interested in me opening a PR to update your template to the latest version of the lib, and rewrite some code to use slash commands rather than old prefix commands ?
